packages/cashier/src/components/crypto-transactions-history/__tests__/crypto-transactions-renderer.spec.tsx

```diff
@@ -52,9 +73,7 @@ describe('<CryptoTransactionsRenderer />', () => {
     });
 
     it('should close the popover when "No" button is clicked in Desktop mode', async () => {
-        const { container } = render(
-            <CryptoTransactionsRenderer row={row} currency={'BTC'} is_transaction_clicked is_valid_to_cancel={1} />
-        );
+        const { container } = renderCryptoTransactionsRenderer();
```

Comment:
Can't `container` be removed?

```suggestion
        renderCryptoTransactionsRenderer();
```